src/azure-cli/azure/cli/command_modules/iotcentral/tests/latest/test_iotcentral_commands.py

```diff
@@ -15,7 +15,7 @@ def test_iotcentral_app(self, resource_group, resource_group_location):
         template_app_display_name = "My Custom App Display Name"
         rg = resource_group
         location = resource_group_location
-        template = 'iotc-pnp-preview'
+        template = 'iotc-pnp-preview@1.0.0'
```

Comment:
it seems several days ago you just recommend not to have the version, ^_^

Reply:
@kawayiboy  These tests run in live mode which looks at the response from rp service. The response in rp service will contain ` template: iotc-pnp-preview@1.0.0 `  which we do an assert against. If we assert `iotc-pnp-preview` against the response from rp service `iotc-pnp-preview@1.0.0` it will always fail.... I'm not sure how these tests passed in the previous PR. My only assumption is the tests weren't ranin `--live` mode and this error slipped through the cracks.

Reply:
@kawayiboy I'm actually pretty sure this is the case because the diffs in this PR says the last time the tests were ran were 2020-01-13 which doesn't correspond to the PR you submitted :)

Reply:
That was me haha. Guess I didn't run the tests with `-live` mode. Good catch there.